src/axom/multimat/examples/calculate.cpp

```diff
@@ -1341,12 +1474,11 @@ void calculate_pressure_cell_dom_full_mm_direct(MultiMat& mm)
     {
       for(int m = 0; m < nmats; ++m)
       {
-        if(*Volfrac.findValue(ic, m) > 0.)
```

Comment:
It's a bit difficult to tell from github -- are we still using the `findValue()` function in any of the experiments? Or were they all converted to the parenthesis operator?

Reply:
[From the PR description]:
> We also standardize on the use of BivariateMap::operator()(firstIdx, secondIdx) for direct-access, since what we really care about is the performance hit from the abstraction.

I saw your comment in the PR description, but was wondering if we should be comparing the difference between the two in any cases.

Reply:
I think the only difference between the two is that `findValue()` returns a pointer to the value, which allows you to check for an invalid index by comparing it with `nullptr`. In the places where we were directly dereferencing this pointer, an invalid index would segfault anyways, so these were just replaced with the parenthesis operator.

But in the "happy" case, both `findValue()` and `operator()` call `bset->getElementFlatIndex(firstIdx, secondIdx)`, and reference into the underlying set with that flat index.